src/main/java/com/jakubroks/quiz/controller/GameController.java

```diff
@@ -1,38 +1,117 @@
 package com.jakubroks.quiz.controller;
 
+import com.jakubroks.quiz.dto.QuestionDTO;
+import com.jakubroks.quiz.dto.QuizResultDTO;
+import com.jakubroks.quiz.entity.SavedGameEntry;
+import com.jakubroks.quiz.entity.User;
 import com.jakubroks.quiz.entry.GameEntry;
 import com.jakubroks.quiz.input.AnswerInput;
 import com.jakubroks.quiz.input.GameInput;
+import com.jakubroks.quiz.repository.SavedGameEntryRepository;
+import com.jakubroks.quiz.service.AuthService;
 import com.jakubroks.quiz.service.GameService;
+import com.jakubroks.quiz.service.ReportService;
+import com.jakubroks.quiz.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
 
 @RestController
 @RequestMapping("/game")
+//@RequiredArgsConstructor
```

Comment:
staraj się nie zostawiać zbędnych komentarzy kodu, a w sumie dlaczego nie może to zostać a usunąć konstruktor (wygląda na to samo)